great_expectations/checkpoint/checkpoint.py

```diff
@@ -1029,6 +1117,19 @@ def run(
         result_format=None,
         **kwargs,
     ):
+        """Legacy Checkpoint that has been deprecated since 0.14.0.
+
```

Comment:
Had to do this one because the linter was complaining. Did not add the `@public_api` decorator